Gems/GradientSignal/Code/Source/GradientSampler.cpp

```diff
@@ -61,45 +61,38 @@ namespace GradientSignal
 
                     ->DataElement(0, &GradientSampler::m_invertInput, "Invert Input", "")
                     ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
-                    ->DataElement(0, &GradientSampler::m_enableTransform, "Enable Transform", "")
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
+
+                    ->GroupElementToggle("Enable Transform", &GradientSampler::m_enableTransform)
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, false)
                     ->DataElement(0, &GradientSampler::m_translate, "Translate", "")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientSampler::AreTransformSettingsDisabled)
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
                     ->DataElement(0, &GradientSampler::m_scale, "Scale", "")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientSampler::AreTransformSettingsDisabled)
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
                     ->DataElement(0, &GradientSampler::m_rotate, "Rotate", "Rotation in degrees.")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientSampler::AreTransformSettingsDisabled)
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
 
-                    ->DataElement(0, &GradientSampler::m_enableLevels, "Enable Levels", "")
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
+                    ->GroupElementToggle("Enable Levels", &GradientSampler::m_enableLevels)
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, false)
```

Comment:
suggestion: This is super minor and might not be worth doing in this case, but this looks technically like the 'usage' code - sometimes it's nice to split the changes in a PR to the new 'feature/API' that's added and then the code that's actually using it. It is likely overkill here but can be useful to keep in mind for future reference.

Reply:
Okay, thanks I'll keep that in mind for future reference!